mods/ra/mod.yaml

```diff
@@ -8,15 +8,16 @@ RequiresMods:
 	modchooser: {DEV_VERSION}
 
 Packages:
-	~^Content/ra
-	~^Content/ra/music
+	~^Content/ra/v2/
+	~^Content/ra/v2/expand
+	~^Content/ra/v2/cnc
 	.
 	$ra: ra
 	$cnc: cnc
 	./mods/common: common
 	~main.mix
-	~redalert.mix
```

Comment:
This makes the game not detect manually copied installations and attempt to download redundant files.